langfuse/serializer.py

```diff
@@ -29,6 +30,8 @@ def default(self, obj: Any):
 
         if is_dataclass(obj):
             return asdict(obj)
+        if isinstance(obj, Sequence) and not isinstance(obj, (str, bytes)):
```

Comment:
In general, i like this approach.
Should this condition come below line 48 `isinstance(obj, (dict, list, str, int, float, type(None)))`?
Also, is it possible that obj is a Sequence and a str at the same time? If no, i think the second condition is not necessary, right?

Reply:
if I move it below L48, then yes we might not need it.

Reply:
I moved it below now @maxdeichmann